src/components/PageLayout/PageLayout.tsx

```diff
@@ -39,24 +38,18 @@ export const PageLayout: FC<PageLayoutProps> = ({contentWidth = 2, title = '', c
       </Head>
       <PageTitleContainer.Provider initialState={title}>
         <BannerContainer.Provider>
-          <div className={styles.pageContainer}>
+          <Stack sx={{minHeight: '100vh', position: 'relative'}}>
             <TopGrid />
-            <MenuMain />
-            <StromLogo />
-            <div className={styles.grid}>
-              <div
-                className={clsx(
-                  styles.mainContent,
-                  contentWidth === 1 && styles.col1,
-                  contentWidth === 2 && styles.col2,
-                  contentWidth === 3 && styles.col3,
-                )}
-              >
+            <Grid container sx={{height: '100%', flex: 1}} columns={{xs: 6, md: 12}}>
```

Comment:
`height: 100%` mi tiez bije do oci, ze to asi nebude ideal.. ale vlastne mas `flex: 1`, to by som asi cakal, ze bude stacit. kludne zrevidujme neskor

Reply:
urcite to bude stacit, ta height je nieco co som viacmenej zabudol zmazat